CONTRIBUTING.md

```diff
@@ -120,213 +122,73 @@ the team support them (though the plugin may work, and will usually install).
-and any other modifications and tasks that you want to include in your builds.  The 
-most important entry is the path to your IDEA installation.  That way there is no
-guessing on the part of `common.xml`, and you don't have to type it on the command
-line all of the time.  Plus, your extra build goodies become available inside of IDEA.
 
-Here is a minimalist example:
+#### IDEA builds
 
-    <project name="local-overrides">
-      <property name="idea.ultimate.build" location="/home/username/intellij_idea/idea-IU-135.1286/" />
-    </project>
+The preferred way for casual developers to build the plugin is using the
+build that they use for their other work.  That is, casual developers shouldn't need to
+be using the command line at all, they should use the normal Build menu commands 
+(or their shortcuts) that they are used to.
 
-Here is a version from one of our team members:
+For these to work with IDEA you will have to tell IDEA to delegate te build job to Gradle
+You should find the option to do this under : Build -> Build tools -> Gradle -> Runner  
 
-    <project name="local-overrides">
-      <echo>
-        Including local overrides...
-      </echo>
-    
-      <property environment="env"/>
-      <property name="idea.installation.dir" location="${env.HOME}/intellij_idea" />
-      <condition property="idea.ultimate.build" value="${idea.installation.dir}/idea-IU-135.1286/" >
-        <matches pattern="13" string="${version}"/>
-      </condition>
-      <condition property="idea.ultimate.build" value="${idea.installation.dir}/idea-IU-141.177.4/">
-        <matches pattern="14.1" string="${version}"/>
-      </condition>
-      <condition property="idea.ultimate.build" value="${idea.installation.dir}/idea-IU-139.1117.1/">
-        <matches pattern="14" string="${version}"/>
-      </condition>
-    
-    </project>
+![gradle delegate](./doc/idea_gradle_delegate.png)
 
-Using this latter file, when started using the command `ant -Dversion=13`, the 
-plugin is built using the IDEA SDK found at `/home/user/intellij_idea/idea-IU-135.1286`.
- 
-##### Generation of META-INF files
-
-Since each version of IDEA has it's own requirements, should not be built
-using code that is specific to another version, and should not be able
-to load into a version of IDEA other than which it was intended, we have 
-to update the src/META-INF files with data appropriate to the version
-being built.  Before compilation, the "metainf" ant task runs and fills 
-in the blanks (well... the areas between @...@ signs) in plugin.xml with 
-the aforementioned values.  The generation algorithm copies the src/META-INF 
-files to gen/META-INF as part of this process, and the project uses the 
-copied files. The properties are defined in properties files in the 
-project root directory (e.g. idea_v17.properties).
 
-#### IDEA builds
+The Gradle Project panel might be a bit overwhelming for those who are not used to Gradle
```

Comment:
Missing period.

Reply:
There are a number of missing periods in this document.  A quick scan and cleanup is warranted.